packages/pangea-sdk/pangea/services/authn/authn.py

```diff
@@ -0,0 +1,252 @@
+                authenticator=authenticator,
+                id_provider=id_provider,
+                verified=verified,
+                require_mfa=require_mfa,
+                profile=profile,
+                scopes=scopes,
+            )
+            response = self.request.post("user/create", data=input.dict(exclude_none=True))
+            if response.raw_result is not None:
+                response.result = UserCreateResult(**response.raw_result)
+            return response
+
+        #   - path: authn::/v1/user/delete
+        # https://dev.pangea.cloud/docs/api/authn#delete-a-user # FIXME: Update url once in prod
+        def delete(self, email: str) -> PangeaResponse[UserDeleteResult]:
+            input = UserDeleteRequest(email=email)
+            response = self.request.post("user/delete", data=input.dict(exclude_none=True))
+            if response.raw_result is not None:
+                response.result = UserDeleteResult(**response.raw_result)
+            return response
+
+        #   - path: authn::/v1/user/invite
+        # https://dev.pangea.cloud/docs/api/authn#invite-a-user # FIXME: Update url once in prod
+        def invite(
+            self,
+            inviter: str,
+            email: str,
+            callback: str,
+            state: str,
+            invite_org: Optional[str] = None,
+            require_mfa: Optional[bool] = None,
+        ) -> PangeaResponse[UserInviteResult]:
+            input = UserInviteRequest(
+                inviter=inviter,
+                email=email,
+                callback=callback,
+                state=state,
+                invite_org=invite_org,
+                require_mfa=require_mfa,
+            )
+            response = self.request.post("user/invite", data=input.dict(exclude_none=True))
+            if response.raw_result is not None:
+                response.result = UserInviteResult(**response.raw_result)
+            return response
+
+        #   - path: authn::/v1/user/list
+        # https://dev.pangea.cloud/docs/api/authn#list-users # FIXME: Update url once in prod
+        def list(self, scopes: Scopes, glob_scopes: Scopes) -> PangeaResponse[UserListResult]:
+            input = UserListRequest(scopes=scopes, glob_scopes=glob_scopes)
+            response = self.request.post("user/list", data=input.dict(exclude_none=True))
+            response.result = UserListResult(**response.raw_result)
+            return response
+
+        #   - path: authn::/v1/user/login
+        # https://dev.pangea.cloud/docs/api/authn#user-login # FIXME: Update url once in prod
+        def login(self, email: str, secret: str, scopes: Optional[Scopes] = None) -> PangeaResponse[UserLoginResult]:
+            input = UserLoginRequest(email=email, secret=secret, scopes=scopes)
+            response = self.request.post("user/login", data=input.dict(exclude_none=True))
+            response.result = UserLoginResult(**response.raw_result)
+            return response
+
```

Comment:
@pangea-andrest should there also be a `authn::/v1/user/update` method here? I have it on my list for the Node SDK, or should I not include it as well?

Reply:
Same question for if we should also include a method for `authn::/v1/userinfo`?

Reply:
My bad. Both should be implemented. here are docs pages 
https://authn.dev.aws.pangea.cloud/v1/user/update
https://authn.dev.aws.pangea.cloud/v1/userinfo
Thanks for notice it!